src/main/java/org/assertj/core/util/Sets.java

```diff
@@ -22,9 +22,22 @@
 /**
  * Utility methods related to {@link Set}s.
  *
- * @author alruiz
+ * @author Alex Ruiz
  */
 public final class Sets {
+
+  /**
+   * Creates a <em>mutable</em> {@link HashSet} containing the given elements.
+   *
+   * @param <T> the generic type of the {@code HashSet} to create.
+   * @param elements the elements to store in the {@code HashSet}.
+   * @return the created {@code HashSet}, or {@code null} if the given array of elements is {@code null}.
+   */
+  @SafeVarargs
+  public static <T> Set<T> set(T... elements) {
+    return newLinkedHashSet(elements);
+  }
```

Comment:
Perhaps I should highlight this addition in the util class, which would be publicly available. Let me know if you have any problems with it.

Reply:
we can but for me these util classes are not meant to be API so we don't need to publicize them.